src/components/profileConf/profileConfig/index.js

```diff
@@ -2,23 +2,16 @@ import React, { useState } from "react";
 import { ProfileConf } from '../index.js';
 import { motion } from "framer-motion";
 import closeModalButton from '../images/cancel.png';
+import { ChanginPass } from "./changingPass/index.js";
```

Comment:
Also, if the folder has the same name as the react component "ChanginPass" (There's a typo) you can just use in the import the following: 
`import { ChangingPass } from "./ChangingPass"` without the index.js because with babel it's implicit